integration/maptest/map.go

```diff
@@ -244,16 +256,18 @@ func RunMapRevisionInvalid(ctx context.Context, t *testing.T, tadmin trillian.Tr
 		},
 	} {
 		for _, hashStrategy := range tc.HashStrategy {
```

Comment:
Consider using `t.Run` here. This would remove the need to introduce a Goto location definition.